BaseUI/src/components/SubscriberCell.tsx

```diff
@@ -0,0 +1,110 @@
+import React, {useState} from 'react';
+import {View, Image, Text} from 'react-native';
+import SubscriberCellProps from '../interfaces/SubscriberCellProps';
+//import SubscriberCellStyle from '../styles/SubscriberCellStyle';
+import FilledButton from './FilledButton';
+// import {CheckBox} from 'react-native-elements';
+// import Icon from 'react-native-vector-icons/FontAwesome';
+// import {CheckBox} from '@rneui/base';
+// import CheckBox from '@react-native-community/checkbox';
+import {CheckBox} from '@rneui/base';
+import {useSwipe} from '../hooks/useSwipe';
+import Colors from '../theme/Colors';
+
+// Icon.loadFont();
+
+const SubscriberCell = (props: SubscriberCellProps) => {
+  //console.log(props.subscriber);
+
+  const [cbState, setCbState] = useState(false);
+
+  function onSwipeLeft() {
+    if (!props.setSubscribers) {
+      return;
+    }
+
+    props.setSubscribers(data => {
```

Comment:
You are in SubscriberCell. It should be responsible for a subscriber but you try to work with all subscribers here.
You can add a callback and name it like 'onSubscriberCellOpened' or something else. And then modify your subscribers on the screen which knows about all of them.